### PR TITLE
Avatar component

### PR DESCRIPTION
**Pull request by dev-prabhat**

### CODE REVIEW

**Review by sanu29 — APPROVED** [review]
All good. File structure, naming convention. They way d-block class u have used is great just a suggestion it should not be in Avtar CSS file but in some other file like utilities or main

**Review by srishtigautam21 — APPROVED** [review]
consistent spacing, good naming of classes.....everything looks fine to me good to merge

File: document/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
```
Comment (bhushannaikaroskar) [inline document/avatar/avatar.css:11]: If you have repeated this part in every file. I would suggest to move to base file of documentation CSS file. Rest is all good

**Review by bhushannaikaroskar — APPROVED** [review]

File: document/avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+.container {
+  width: 90%;
+  margin: 0 auto;
+}
+
+.container-fluid {
+  padding: 0.5em;
+  box-shadow: 0px 0px 15px 0px var(--primary-theme-light-color);
```
Comment (sudipkundu999) [inline document/avatar/avatar.css:20]: when you're assigning 0 to any property you don't need to mention the units

**Review by sudipkundu999 — APPROVED** [review]
Good Work.
Good to merge!

**Pull request merged**